lib/pure/collections/tables.nim

```diff
@@ -960,16 +963,14 @@ proc mget*[A](t: var CountTable[A], key: A): var int {.deprecated.} =
   ctget(t, key)
 
 proc getOrDefault*[A](t: CountTable[A], key: A): int =
-  ## retrieves the value at ``t[key]`` iff ``key`` is in ``t``. Otherwise, 0 (the
+  ## retrieves the value at ``t[key]`` if ``key`` is in ``t``. Otherwise, 0 (the
```

Comment:
Any particular reason you change `iff` to `if`? "iff" reads as "if and only if".

Reply:
no reason at all, will change it back.

Reply:
Looks like you accidentally changed it back to `if` in the last commit, both here and below